app/src/main/java/com/fastlink/zekrapp/data/data/CategoryData.kt

```diff
@@ -0,0 +1,935 @@
+package com.fastlink.zekrapp.data.data
+
+val categoryData : ArrayList<ArrayList<Any>> = arrayListOf(
```

Comment:
Naming it zekirType or zekirCategory is clearer than having just category, if we read it as just category we don't know what is it.